docs/index.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en-us">
+<head>
+  <meta charset="utf-8">
+  <title>TMAPI</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+  <script src="../dist/ticketmaster-client-1.0.0.min.js"></script>
+  <script>
+    (function(TMAPI) {
+      function logPage(page) {
+        console.log(page.page.number, page.items.map((item) => item.name))
+
+        return page.getNext()
+      }
+
+      const sdk = TMAPI.default('pTAf3X1DA6XCHjaQaw0WmlgcA0j1r32C')
```

Comment:
Is there a way we can remove this API key from the source, yet have it be clear to the user that this `docs/index.html` file requires you to make that change locally?